templates/upstart.conf.j2

```diff
@@ -0,0 +1,14 @@
+description "Gunicorn upstart script for {{ gunicorn_app_name }}"
+
+{% if develop or test -%}
```

Comment:
Right now the default is the `local-filesystems` one, but in my next commit I'm changing the example to use `vagrant-mounted`.